socrates/lib/site/views/schedule.jade

```diff
@@ -20,34 +20,66 @@ block content
           +extlink('https://en.wikipedia.org/wiki/The_World_Cafe', 'World Café')
-          | #{''} in which we all discuss the question "What do you want to do at
-          | SoCraTes?".
+          | , to help kickstart everyone into a great conference experience.
         dt Friday
-        dd Open Space day 1. We start at 9.30h and there will be six time slots
-          | for you to fill with your ideas, sessions, workshops, questions and
-          | whatever else you'd like to bring or get from SoCraTes.
+        dd Friday is completely reserved for a full-day #{''}
+          +extlink('http://agilecoachcamp.org/tiki-index.php?page=OpenSpace', 'Open Space')
+          | .
+        dd.
+          We start the marketplace at 9am and the first sessions will start
+          around 10.30am. The last session timeslot ends at 6pm, which is also
+          the time we all meet again in the marketplace room for a joint
+          retrospective of the day.
+        dd
+          | Around 7pm dinner is available for around 2h, so everyone can start
+          | into the evening at their own pace. Of course there's lots of room for
+          | shared #{''}
+          a(href='/wiki/2015/evening-activities') evening activities
+          |  like board games, making music, hacking or just talking and having fun.
         dt Saturday
-        dd Open Space day 2. The second day provides opportunities in form of
-          | five time slots for the participants to fill.
-        dt Sunday (optional, you can choose to attend when registering)
-        dd Workshops, e.g. a #{''}
-          +extlink('http://coderetreat.org/', 'Coderetreat')
-          |  (details to be announced).
+        dd Saturday is also completely reserved for a full-day #{''}
+          +extlink('http://agilecoachcamp.org/tiki-index.php?page=OpenSpace', 'Open Space')
+          | .
+        dd.
+          We start the marketplace again at 9am and the first sessions will
+          start around 10.30am too. The last session timeslot ends at 4.30pm.
+          After that we again all meet for the closing of the conference and a
+          retrospective of the two days. The closing of the conference will most
+          likely end around 5.30pm.
+        dd Participants staying until Sunday (or longer) can have dinner starting at 7pm.
+        dt Sunday
+        dd
+          | Sunday is reserved for workshop oriented formats. Go and make a
+          | suggestion or request for a workshop on the #{''}
+          a(href='/wiki/2015/sunday-workshops') Sunday Workshops
+          |  page.
+        dd.
+          Each workshop can choose themselves when they want to start (though
+          rooms are available at 9am) and when they want to end, but the latest
+          they have to end sometime around 6pm.
       p
         | To get an impression of what you can expect, have a look at 2013's schedule for #{''}
```

Comment:
"schedule" might be confusing here. Perhaps rather "Open Space timetable" or something?